test/e2e/testsuite/testsuite.go

```diff
@@ -253,6 +279,46 @@ func InitializeScenario(s *godog.ScenarioContext) {
 					fmt.Println(err)
 					os.Exit(1)
 				}
+
+				err = crcCmd.UnsetConfigPropertySucceedsOrFails("preset", "succeeds") // unsetting property that is not set gives 0 exitcode, so this works
+				if err != nil {
+					fmt.Println(err)
+					os.Exit(1)
+				}
+			}
+
+			if tag.Name == "@proxy" {
+
+				err := util.ExecuteCommand("podman stop squid")
+				if err != nil {
+					fmt.Println(err)
+					os.Exit(1)
+				}
+
+				err = util.ExecuteCommand("podman rm squid")
+				if err != nil {
+					fmt.Println(err)
+					os.Exit(1)
+				}
+
+				err = crcCmd.UnsetConfigPropertySucceedsOrFails("http-proxy", "succeeds") // unsetting property that is not set gives 0 exitcode, so this works
+				if err != nil {
+					fmt.Println(err)
+					os.Exit(1)
+				}
+
+				err = crcCmd.UnsetConfigPropertySucceedsOrFails("https-proxy", "succeeds") // unsetting property that is not set gives 0 exitcode, so this works
+				if err != nil {
+					fmt.Println(err)
+					os.Exit(1)
+				}
+
+				// crc oc-env sets these behind quietly if http(s)-proxy is set in crc config
+				err = util.ExecuteCommand("unset HTTP_PROXY HTTPS_PROXY NO_PROXY") // unsetting property that is not set gives 0 exitcode, so this works
```

Comment:
Could this use https://pkg.go.dev/os#Unsetenv instead?

Reply:
I changed it as you suggest, but then I have to do it 3x (`os.Unsetenv` seems to take 1 arg only, and it cannot be multiple vars in 1 string). It functions OK though, I think.